services/card_service.rb

```diff
@@ -0,0 +1,60 @@
+class CardService
+  extend InputHelper
+  extend OutputHelper
+  extend Uploader
+
+  class << self
+    def create_card(accounts, current_account)
+      current_account.card = current_account.card << create_new_card
+      save_data(accounts)
+    end
+
+    def create_new_card
+      loop do
+        card_type = ask_card_type
+        return Object.const_get("#{card_type.capitalize}Card").new if CARD_TYPES.include?(card_type)
+
+        puts I18n.t(:wrong_card_type)
+      end
+    end
+
+    def show_cards(current_account)
+      if current_account.card.any?
+        current_account.card.each { |card| puts "- #{card.number}, #{card}" }
```

Comment:
```suggestion
        current_account.card.each { |card| puts "- #{card.number}, #{card.type}" }
```
please, add type of card for all puts

Reply:
Done